kura/org.eclipse.kura.core.net/src/main/java/org/eclipse/kura/core/net/NetworkConfiguration.java

```diff
@@ -424,9 +422,9 @@ public boolean isValid() throws KuraException {
             }
 
             NetInterfaceType type = netInterfaceConfig.getType();
-            if (type != NetInterfaceType.ETHERNET && type != NetInterfaceType.WIFI
+            if (type != NetInterfaceType.ETHERNET && type != NetInterfaceType.WIFI && type != NetInterfaceType.MODEM
                     && type != NetInterfaceType.LOOPBACK) {
-                s_logger.error("Type must be ETHERNET, WIFI, or LOOPBACK - type is " + type);
+                s_logger.error("Type must be ETHERNET, WIFI, MODEM, or LOOPBACK - type is " + type);
```

Comment:
As far as I understand this part of code you usually shouldn't see this. This is called only if you prepare a NetInterfaceAddressConfig for the unexpected type which I would assume doesn't happen very often :)